### PR TITLE
Bootstrap form

### PR DESCRIPTION
**Pull request by pJeyakumar**
Add the `bootstrap_form` gem and apply `bootstrap_form_with` to the HTML form pages.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -7,8 +7,6 @@
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet">
```
Comment (hvillero) [inline app/views/layouts/application.html.erb:10]: 👍

**Review by hvillero — APPROVED** [review]

**Review by srb- — APPROVED** [review]
:+1:

File: app/views/logbook_entries/_form.html.erb [hunk 2]
```diff
@@ -10,124 +10,67 @@
...
   <% end %>
 
-  <div class="mb-3">
-    <%= form.label :date, class: "form-label" %>
-    <%= form.datetime_field :date, class: "form-control", required: true %>
-    <div class="invalid-feedback">
-      Please provide a valid date.
-    </div>
+  <%= form.datetime_field :date, label: "Date", required: true %>
+  <div class="invalid-feedback">
```
Comment (jonathanloos) [inline app/views/logbook_entries/_form.html.erb:14]: Not sure this is needed?

**Review by jonathanloos — CHANGES_REQUESTED** [review]
Couple small things!

File: app/views/logbook_entries/_form.html.erb (hunk 2 above)
Comment (pJeyakumar) [reply app/views/logbook_entries/_form.html.erb:14]: Nope it is not needed, looks like I forgot to remove that

**Review by jonathanloos — APPROVED** [review]

**Pull request merged**